### PR TITLE
Artisan command to load in pre-Gambit (or any!) signups from CSV

### PR DESCRIPTION
**Pull request by katiecrane**
#### What's this PR do?
Includes a script to import signups from a csv! The csv will include ALL pre-Gambit signups. The script checks if a signup already exists for the given user, campaign, and campaign run before creating a new one so there are no duplicates.

#### How should this be reviewed?
Will this run right?

#### Any background context you want to provide?
This is gonna make Rogue the source of truth!!

#### Relevant tickets
[Pivotal](https://www.pivotaltracker.com/story/show/151064896)

#### Checklist
- [ ] Tested on staging.

### CODE REVIEW

File: app/Console/Commands/ImportSignupsCommand.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(Registrar $registrar)
+    {
+        parent::__construct();
+
+        $this->registrar = $registrar;
```
Comment (DFurnes) [inline app/Console/Commands/ImportSignupsCommand.php]: It looks like this might not be used!

File: app/Console/Commands/ImportSignupsCommand.php [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            // Create a signup if there isn't on already
+            if (! $existing_signup) {
+                $signup = new Signup;
+                $signup->northstar_id = $missing_signup['northstar_id'];
+                $signup->campaign_id = $missing_signup['campaign_node_id'];
+                $signup->campaign_run_id = $missing_signup['campaign_run_id'];
+                $signup->source = 'sms';
+                $signup->created_at = $missing_signup['signup_created_at_timestamp'];
+                $signup->updated_at = Carbon::now();
+                $signup->save(['timstamps' => false]);
```
Comment (DFurnes) [inline app/Console/Commands/ImportSignupsCommand.php]: `timstamps` → `timestamps` 👀

Comment (katiecrane) [reply app/Console/Commands/ImportSignupsCommand.php]: hahahhahahha

Comment (katiecrane) [reply app/Console/Commands/ImportSignupsCommand.php]: the tests still pass though it looks like it still works 😂

File: app/Console/Commands/ImportSignupsCommand.php (hunk 1 above)
Comment (katiecrane) [reply app/Console/Commands/ImportSignupsCommand.php]: oh it sure isn't. old plans.

**Pull request merged**